src/shaders/_prelude_terrain.vertex.glsl

```diff
@@ -31,6 +36,10 @@ float decodeElevation(vec4 v) {
 }
 
 float currentElevation(vec2 apos) {
+#ifdef DEM_FLOAT_FORMAT
```

Comment:
Should we be consistent with the other define and name that TERRAIN_DEM_FLOAT_FORMAT? (we already have TERRAIN_DEM_NEAREST_FILTER)

Reply:
At first, I didn't follow it because it is used in hillshade too, without terrain. Renamed to TERRAIN_DEM_FLOAT_FORMAT - it is terrain DEM anyway.